docs/extend/modular-resources/_index.md

```diff
@@ -279,6 +282,55 @@ func init() {
 }
 ```
 
+<file>mymodule.go</file>
+
+``` go {class="line-numbers linkable-line-numbers"}
+// Package main is a module which serves the mybase custom model type.
+package main
+
+import (
+    "context"
+
+    "github.com/edaniels/golog"
+
+    "go.viam.com/rdk/components/base"
+    "go.viam.com/rdk/module"
+    "go.viam.com/utils"
+
+    // NOTE: You must update the following line to import your local package "mybase"
```

Comment:
@penguinland LMK if this does not provide enough info here-- would this be the path to mybase after compiling?

Reply:
"After compiling" doesn't really make sense. It's the path to mybase, full stop. I like the comment: it's saying "you probably changed the package name when you made your own version of the code, so remember to change this line to match."